cmd/snap-recovery-chooser/main.go

```diff
@@ -119,6 +137,15 @@ func cleanupTriggerMarker() error {
 }
 
 func chooser(cli *client.Client) error {
+	snappyTesting := os.Getenv("SNAPPY_TESTING") != ""
```

Comment:
won't this change the behavior during spread tests generally? Or perhaps this binary will only ever be used in spread in an artificial way?

Reply:
ah, didn't notice setting this top level in spread.yaml :disappointed: